lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -651,6 +827,7 @@ def __init__(self, config):
         self.scratch = config['scratch']
         self.callback_url = os.environ['SDK_CALLBACK_URL']
         self.ws_url = config['workspace-url']
+        self.token_user = os.environ['KB_AUTH_TOKEN'].split('client_id=')[1].split('|')[0]
```

Comment:
@MrCreosote Thanks! using ctx['token']. By the way, in my local test, ctx['user_id'] is None. So I am still using parsing user_name from token.